projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/read-case-flag-field.component.spec.ts

```diff
@@ -387,6 +387,7 @@ describe('ReadCaseFlagFieldComponent', () => {
     fixture = TestBed.createComponent(ReadCaseFlagFieldComponent);
     component = fixture.componentInstance;
     component.caseField = flagLauncher1CaseField;
+    component.formGroup = new FormGroup({});
```

Comment:
@MateiHMCTS - is this strictly required? It seems the unit tests that need one initialize `component.formGroup` with the `const formGroup` that is declared earlier

Reply:
Required for the change in [read-case-flag-field.component.ts](https://github.com/hmcts/ccd-case-ui-toolkit/pull/1361/files#diff-d946ca96bcd1ed92266e51ef8ae71957f7c31365e40954cd37aa311909568047) -- lines 41-44

Reply:
Thanks for checking, @MateiHMCTS - happy to mark as resolved